lib/node_modules/@stdlib/array/full/test/test.js

```diff
@@ -185,6 +187,22 @@ tape( 'the function returns a filled array (dtype=float32)', function test( t )
 	t.end();
 });
 
+tape( 'the function returns a filled array (dtype=bool)', function test( t ) {
+	var expected;
+	var arr;
+	var v;
+
+	expected = new Uint8Array( [ true, true, true, true ] );
+
+	v = true;
+	arr = full( 4, v, 'bool' );
```

Comment:
```suggestion

	expected = new Uint8Array( [ 1, 1, 1, 1 ] );

	arr = full( 4, true, 'bool' );
```

While `true` values cast to `1`s, I don't see any reason to require that implicit understanding here.